x-pack/plugins/observability_solution/slo/public/embeddable/slo/overview/slo_embeddable_factory.tsx

```diff
@@ -165,11 +165,21 @@ export const getOverviewEmbeddableFactory = ({
               const kqlQuery = groupFilters?.kqlQuery ?? '';
               const groups = groupFilters?.groups ?? [];
               return (
-                <Wrapper>
+                <div
+                  css={({ euiTheme }: UseEuiTheme) => css`
+                    width: 100%;
+                    padding: ${euiTheme.size.xs} ${euiTheme.size.base};
+                    overflow: scroll;
+
+                    .euiAccordion__buttonContent {
```

Comment:
might be something we can apply directly in the acordion in group list view.

Reply:
I'd like to keep the goal of this PR solely on removing the deprecated theme provider and migrating to emotion.